src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-340.mdx

```diff
@@ -7,9 +7,9 @@ downloadLink: 'https://play.google.com/store/apps/details?id=com.newrelic.rpm'
 
 ### Notes
 
-* This release adds support for multi-query widgets in dashboards.
+* This release adds support for multi query widgets in dashboards.
 
 ### Fixes
 
-* Fixes crash in NRQL Editor
+* Fixes crash in Nrql Editor
```

Comment:
@lwegener - The style is all upper case for NRQL, since it's an acronym.  Was there a reason you downcased it?

Reply:
nope, just ignorance